### ISSUE TITLE
Add magic.link authentication integration

### ISSUE THREAD

**Issue by samjbmason** [gahabeen/biota#34]
I know they’re not as big as Facebook or github but they are a great passwordless alternative

**Comment by gahabeen** [comment]
Of course. Have you ever used it?
I'll add the OAuth2 helpers in the source today.
I've done so far Google and Github as examples. Then it pretty much copy/paste for each provider.

**Comment by samjbmason** [comment]
Yeah I have used it in a little side project. It doesn’t use oauth so not
sure if that will change much but hopefully it shouldn’t be too complex to
integrate.
